helios-testing/src/main/java/com/spotify/helios/testing/TemporaryJobs.java

```diff
@@ -242,8 +242,6 @@ public TemporaryJobBuilder job() {
    * </ol>
    *
    * @return an instance of TemporaryJobs
-   * @see <a href="https://github.com/spotify/helios/blob/master/docs/testing_framework.md#
-   * configuration-by-file">Helios Testing Framework - Configuration By File</a>
    */
```

Comment:
Could we instead change the `@see` tag to this? Formatting looks a little funny, but it passes the javadoc check.

    * @see <a href=
    * "https://github.com/spotify/helios/blob/master/docs/testing_framework.md#configuration-by-file"
    * >Helios Testing Framework - Configuration By File</a>